InvenTree/InvenTree/settings.py

```diff
@@ -839,8 +839,18 @@
         logger.warning(f"The custom logo file '{CUSTOM_LOGO}' could not be found in the static or media directories")
         CUSTOM_LOGO = False
 
+# Check for a custom splash screen
+CUSTOM_SPLASH = get_setting('INVENTREE_CUSTOM_SPLASH', 'customize.splash', None)
+
+if CUSTOM_SPLASH:
+    if static_storage.exists(CUSTOM_SPLASH):
+        logger.info(f"Loading custom splash screen from static directory: {CUSTOM_SPLASH}")
+    else:
+        logger.warning(f"The custom splash screen '{CUSTOM_SPLASH}' could not be found in the static directory")
+        CUSTOM_SPLASH = False
```

Comment:
Suggestion: Maybe this section could be refactored together with CUSTOM_LOGO to remove duplication?

Reply:
Could do, I'll leave that for a future PR though